docs/integration/java/firezone.md

```diff
@@ -0,0 +1,52 @@
+---
+title: FireZone
+description:  Using OIDC protocol as IDP to connect various applications, like FireZone
+keywords: [OIDC, FireZone, IDP]
+author: dingchenzy
+---
+
+[Casdoor](/docs/basic/server-installation) can use OIDC protocol as IDP to connect various applications. Here we will use [FireZone](https://docs.firezone.dev/deploy/) as an example to show you how to use OIDC to connect to your applications.
+
+# Step 1. Deploy Casdoor and FireZone
```

Comment:
![image](https://user-images.githubusercontent.com/53544726/202692941-f6239dc1-5f94-4caf-8eff-d4083eb9f79c.png)
https://github.com/casdoor/casdoor-website#documentation-writing
h1 is not recommended

Reply:
@dingchenzy